html/elements/style.json

```diff
@@ -0,0 +1,281 @@
+                  },
+                  "opera": {
+                    "version_added": "3.5"
+                  },
+                  "opera_android": {
+                    "version_added": "6.0"
+                  },
+                  "safari": {
+                    "version_added": "1.0"
+                  },
+                  "safari_ios": {
+                    "version_added": "1.0"
+                  }
+                },
+                "status": {
+                  "experimental": false,
+                  "standard_track": true,
+                  "deprecated": false
+                }
+              }
+            }
+          },
+          "title": {
+            "__compat": {
+              "basic_support": {
+                "support": {
+                  "webview_android": {
+                    "version_added": "1.0"
+                  },
+                  "chrome": {
+                    "version_added": "1.0"
+                  },
+                  "chrome_android": {
+                    "version_added": "1.0"
+                  },
+                  "edge": {
+                    "version_added": true
+                  },
+                  "edge_mobile": {
+                    "version_added": true
+                  },
+                  "firefox": {
+                    "version_added": "1.0"
+                  },
+                  "firefox_android": {
+                    "version_added": "1.0"
+                  },
+                  "ie": {
+                    "version_added": "3.0"
+                  },
+                  "ie_mobile": {
+                    "version_added": "9.0",
+                    "notes": "Mobile Internet Explorer (the previous branding of IE Phone - versions lower than 8) also has support."
+                  },
+                  "opera": {
+                    "version_added": "3.5"
+                  },
+                  "opera_android": {
+                    "version_added": "6.0"
+                  },
+                  "safari": {
+                    "version_added": "1.0"
+                  },
+                  "safari_ios": {
+                    "version_added": "1.0"
+                  }
+                },
+                "status": {
+                  "experimental": false,
+                  "standard_track": true,
+                  "deprecated": false
+                }
+              }
+            }
+          },
+          "scoped": {
+            "__compat": {
+              "basic_support": {
+                "support": {
+                  "webview_android": {
+                    "version_added": false
+                  },
+                  "chrome": {
+                    "version_added": "19.0",
+                    "version_removed": "35.0",
+                    "flag": {
+                      "type": "preference",
+                      "name": "Enable <style scoped>",
```

Comment:
Fixed. Will be in the next rebased PR.

Note that you can't automate escaping as &lt;code&gt; and &lt;a&gt; are ambiguous.